webapp/channels/src/components/widgets/menu/menu_items/restricted_indicator.tsx

```diff
@@ -107,27 +110,29 @@ const RestrictedIndicator = ({
                 )}
             >
                 {useModal && blocked ? (
-                    <ToggleModalButton
-                        id={`${feature}-restricted-indicator`.replaceAll('.', '_')}
-                        className='RestrictedIndicator__button'
-                        modalId={ModalIdentifiers.FEATURE_RESTRICTED_MODAL}
-                        dialogType={FeatureRestrictedModal}
-                        onClick={handleClickCallback}
-                        dialogProps={{
-                            titleAdminPreTrial,
-                            messageAdminPreTrial,
-                            titleAdminPostTrial,
-                            messageAdminPostTrial,
-                            titleEndUser,
-                            messageEndUser,
-                            customSecondaryButton: customSecondaryButtonInModal,
-                            feature,
-                            minimumPlanRequiredForFeature,
-                        }}
-                    >
-                        {icon}
-                        {ctaExtraContent}
-                    </ToggleModalButton>
+                    <span>
```

Comment:
This span is needed because Bootstrap tooltip needs a native element in the same component for it to work and Its may not be able to find the target element it's supposed to attach to due to internal working of Bootstrap Tooltip